Tests/TuistAutomationTests/Utilities/BuildGraphInspectorTests.swift

```diff
@@ -65,21 +65,38 @@ final class BuildGraphInspectorTests: TuistUnitTestCase {
         let target = Target.test(platform: .tvOS)
 
         // When
-        let got = subject.buildArguments(target: target, configuration: nil)
+        let got = subject.buildArguments(target: target, configuration: nil, skipSigning: false)
 
         // Then
         XCTAssertEqual(got, [
             .sdk(Platform.tvOS.xcodeSimulatorSDK!),
         ])
     }
 
+    func test_buildArguments_when_skipSigning() {
+        // Given
+        let target = Target.test(platform: .iOS)
+
+        // When
+        let got = subject.buildArguments(target: target, configuration: nil, skipSigning: true)
+
+        // Then
+        XCTAssertEqual(got, [
+            .sdk(Platform.iOS.xcodeSimulatorSDK!),
```

Comment:
you can use `try XCTUnwrap(...)` here, so the test does not crash here if it fails the unwrapping

Reply:
Fixed! Here and in other test cases inside this class